starter-code/pizza.js

```diff
@@ -1 +1,64 @@
 // Write your Pizza Builder JavaScript in this file.
+
+'use strict'
+
+function main(){
+
+    var sauce = '$0 regular sauce';
+    var crust = '$0 regular crust';
+
+    var $peppers = $('.green-pepper');
+    var $peppersButton = $('.btn-green-peppers');
+    var $pepperList = $('.price li:nth-child(1)');
+        $peppersButton.click(function(){
+            $peppersButton.toggleClass('active')
+            $peppers.toggle(function(){
```

Comment:
Why are you passing an empty function? It is doing nothing. Same issue with pepperoni and mushroms.